### PR TITLE
[237] Delete Node in a Linked List

### PR DESCRIPTION
**Pull request by GeorgeNav**
image

### CODE REVIEW

**Comment by GeorgeNav** [comment]
image

File: src/complete/237.delete-node-in-a-linked-list.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ *
+ * [237] Delete Node in a Linked List
+ */
+
+// Definition for singly-linked list.
+class ListNode {
+  val: number
+  next: ListNode | null
+  constructor(val?: number, next?: ListNode | null) {
+    this.val = (val === undefined ? 0 : val)
```
Comment (nmyers217) [inline src/complete/237.delete-node-in-a-linked-list.ts:12]: Does the null coalescing operator work for this?
```suggestion
    this.val = val ?? 0
```
Same thing on the next line.

**Review by nmyers217 — APPROVED** [review]
Just wait until we have to reverse the linked list

File: src/complete/237.delete-node-in-a-linked-list.ts (hunk 1 above)
Comment (GeorgeNav) [reply src/complete/237.delete-node-in-a-linked-list.ts:12]: The code for this was autogenerated by the initial problem code btw. Maybe they wanted to be explicit for some reason. Ya that’s what I would probably do if I wrote it. I think so.

**Pull request merged**